AppCenter/AppCenter/Internals/Context/AuthToken/MSAuthTokenHistoryState.h

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * Represents token data entity.
+ */
+@interface MSAuthTokenHistoryState : NSObject
```

Comment:
Add account id to history

Reply:
We have it in history (`MSAuthTokenInfo` class); Even though the name (`MSAuthTokenHistoryState`) stands for history, it is actually a representation of token validity information and account id is not to be placed here. I'd rather suggest renaming class to smth like `MSAuthTokenValidityInfo`